mongo/datadog_checks/mongo/collectors/replica.py

```diff
@@ -88,6 +92,27 @@ def _report_replica_set_states(self, members, replset_name):
                 event_payload['host'] = self.hostname
             self.check.event(event_payload)
 
+    def get_replset_config(self, client):
+        """On most nodes, simply runs `replSetGetConfig`.
+        Unfortunately when the agent is connected to an arbiter, running the `replSetGetConfig`
+        raises authentication errors. And because authenticating on an arbiter is not allowed, the workaround
+        in that case is to run the command directly on the primary."""
+
+        if self.check.deployment.is_arbiter:
+            try:
+                # Authenticates if the config specifies a username;
+                do_auth = self.check.username is not None
+                cli_primary = self.check.setup_connection(do_auth, replicaset=self.check.deployment.replset_name)
+            except Exception:
+                self.log.warning(
+                    "Current node is an arbiter, the extra connection to the primary was unsuccessful."
+                    " Votes metrics won't be reported."
+                )
+                return None
+            return cli_primary['admin'].command('replSetGetConfig')
```

Comment:
could this raise an exception too? Maybe its safer moving it to inside the `try` block